lib/tasks/watch.js

```diff
@@ -20,6 +23,28 @@ var watchGlobs = [
 	'!**/tmp-src.scss'
 ];
 
+function reportTaskComplete(taskResult, builtExtensions) {
+
+	checkTaskStatus(taskResult, function(err, result) {
+
+		var notificationTitle = "Build succeeded";
```

Comment:
The messages are only valid for the build task, not for demo or test which are the other watchable tasks